cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -139,8 +157,10 @@ _kubectl patch validatingwebhookconfiguration operator-webhook-config --patch '{
 _kubectl patch mutatingwebhookconfiguration network-resources-injector-config --patch '{"webhooks":[{"name":"network-resources-injector-config.k8s.io", "clientConfig": { "caBundle": "'"$(cat $CSRCREATORPATH/network-resources-injector.cert)"'" }}]}'
 _kubectl patch mutatingwebhookconfiguration operator-webhook-config --patch '{"webhooks":[{"name":"operator-webhook.sriovnetwork.openshift.io", "clientConfig": { "caBundle": "'"$(cat $CSRCREATORPATH/operator-webhook.cert)"'" }}]}'
 
-# we need to sleep to wait for the configuration above the be picked up
-sleep 60
+# Ensure caBundle is configured with new certificates
+ensure_caBundle "validatingwebhookconfiguration" "operator-webhook-config"           "$(cat $CSRCREATORPATH/operator-webhook.cert)"
```

Comment:
I'm not sure this will be enough. This only waits for the caBundle to be set, that should happen as soon as we _patch. The timeout was here mostly to wait until the changed caBundle is picked up and used by the webhook. I think this change is not needed. That being said, I don't have a good way how to check for webhook readiness to serve.

Reply:
In line 217 we check if caBundle is picked up by waiting for node taint `NoShcedule` to be removed,
as soon as you patch those webhooks the taint is there, I think its enough because once the taint is removed all the pods are ready, what you think?

ensure_caBundle verifies that caBundle got the right hash, I understand why it my seem redundant

Reply:
Please drop it, let's just trust `kubectl patch` and reduce the amount of code that can break (and we need to maintain)

Reply:
ack